content/about/contributing/contributing.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Contributing to the APG</title>
+
+  <!-- Core JS and CSS shared by all About content -->
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="../../shared/css/core.css">
+  <script src="../../shared/js/highlight.pack.js"></script>
+  <script src="../../shared/js/app.js"></script>
+  <script src="../../shared/js/skipto.js"></script>
+</head>
+
+<body>
+  <main>
+    <h1>Contributing to the APG</h1>
+
+    <section id="intro">
+      <h2>Introduction  </h2>
+      <p>
+        Continuous improvement to the utility and quality of the APG depends on broad community input and participation.
+        Anyone may contribute feedback, suggestions, or code via the channels described below.
+        However, if you would like to contribute regularly, please consider
+        <a href="#joining-task-force">joining the Authoring Practices Task Force</a>.
+        As a task force participant, you have greater influence over APG development, and you and your organization will be listed as contributors where appropriate.
+      </p>
+    </section>
+
+    <section id="feedback">
+      <h2>Providing feedback</h2>
+      <p>
+        The APG provides two public feedback channels -- GitHub and email.
+        Links to both are provided at the bottom of every page in the APG.
+      </p>
+      <p>
+        GitHub issues are the best way to ask a question, make a suggestion,  or report a problem.
+        Before creating a new issue, it is helpful if you first review existing issues to determine whether there is an issue addressing the same concern.
+        If so, please comment on that issue rather than raising another.
+        If the concern you would like addressed is related to an example implementation of an APG pattern, at the bottom of the page, next to the revision date, there is a link named "Related Issues".
+        That link opens a GitHub project containing all issues associated with the relevant pattern and its examples.
+      </p>
+      <p>
+        <a href="https://github.com/w3c/aria-practices/issues">View and create APG issues on GitHub.</a>
+      </p>
+      <p>
+        The task force also uses the public aria-practices mailing list for email discussion.
+        Meeting announcements, agendas, and links to minutes are sent to the mailing list.
+        While GitHub issues are the preferred place to discuss APG content, the mailing list is available to anyone who would prefer to communicate with the APG Task Force by
+        <a href="mailto://public-aria-practices@w3.org">sending email</a>.
+      </p>
+      <p>
+        <a href="https://lists.w3.org/Archives/Public/public-aria-practices/">View the aria-practices mailing list archive</a>.
+      </p>
+    </section>
+    
+    <section id="making-content-improvements">
```

Comment:
```suggestion
    <section id="Making-improvements-to-documentation-or-code">
```

Text to lowercase and hyphenated